Client/src/store/slices/PlaySessionsSlice.ts

```diff
@@ -0,0 +1,45 @@
+import { Slice, createSlice } from "@reduxjs/toolkit";
+import { UserPlaySession } from "../../types/storeSliceTypes";
+
+export const playSessionsSlice: Slice<UserPlaySession[]> = 
+createSlice({
+  initialState: Array<UserPlaySession>,
+  name: "playSessionsSlice",
+  reducers: {
+    setPlaySessions:(state:any,action:any) =>
+    {
+      const data:UserPlaySession[] = action.payload;
+      const retrievedSessions:UserPlaySession[] = data.filter((c:any) => !state?.some((i:any) => i.id === c.id));
```

Comment:
`(c:any) => !state?.some((i:any) => i.id === c.id)` with such names and any type i don't understand anything